### PR TITLE
[EI-909] Add "buildMessage" attribute to management console

### PR DESCRIPTION
**Pull request by Bhashinee**
Fix the issue of not supporting "buildMessage" attribute in the management console.

### CODE REVIEW

File: modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java [hunk 1]
```diff
@@ -50,6 +50,10 @@ protected OMElement serializeEndpoint(Endpoint endpoint) {
 
         serializeCommonAttributes(endpoint,endpointElement);
 
+        if (failoverEndpoint.isBuildMessageAtt()) {
+            failoverElement.addAttribute("buildMessage", Boolean.toString(failoverEndpoint.isBuildMessageAtt()), null);
```
Comment (chanikag) [inline modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java]: As "buildMessage" has been used multiple places, shall we define it as a constant?

Comment (Bhashinee) [reply modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java]: Fixed

File: modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointSerializer.java [hunk 2]
```diff
@@ -37,6 +37,7 @@
  */
 public class SALoadbalanceEndpointSerializer extends EndpointSerializer {
 
+    private static final String BUILD_MESSAGE = "buildMessage";
```
Comment (chanikag) [inline modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointSerializer.java]: Can't we use the same constant defined in XMLConfigConstants?

File: modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointFactory.java [hunk 3]
```diff
@@ -121,6 +122,15 @@ protected Endpoint createEndpoint(OMElement epConfig, boolean anonymousEndpoint,
                     createLoadbalanceAlgorithm(loadbalanceElement, endpoints);
             loadbalanceEndpoint.setAlgorithm(algorithm);
 
+            //set buildMessage attribute
+            String  buildMessageAtt = loadbalanceElement.getAttributeValue(new QName("buildMessage"));
```
Comment (chanikag) [inline modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointFactory.java]: Same applies here. Can we use the constant defined in XMLConfigConstants?

Comment (Bhashinee) [reply modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointFactory.java]: Done

File: modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointSerializer.java (hunk 2 above)
Comment (Bhashinee) [reply modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointSerializer.java]: Done

File: modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java [hunk 4]
```diff
@@ -50,6 +51,10 @@ protected OMElement serializeEndpoint(Endpoint endpoint) {
 
         serializeCommonAttributes(endpoint,endpointElement);
 
+        if (failoverEndpoint.isBuildMessageAtt()) {
+            failoverElement.addAttribute(XMLConfigConstants.BUILD_MESSAGE, Boolean.toString(failoverEndpoint.isBuildMessageAtt()), null);
```
Comment (prabushi) [inline modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java]: Is line 55 in 120 limit?

Comment (Bhashinee) [reply modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java]: Fixed in https://github.com/wso2/wso2-synapse/pull/892

**Pull request merged**